app/lib/commits_populator.rb

```diff
@@ -142,5 +148,18 @@ def back_commit_redis_key
       # this key should refer to the OLDEST commit we have in the db
       "discourse-github-back-commit-#{@repo.name}"
     end
+
+    def disable_github_badges_and_inform_admin
+      SiteSetting.github_badges_enabled = false
+      site_admin_username = User.where(admin: true).human_users.limit(1).pluck(:username).first
```

Comment:
I would send this to all the admins maybe for safety pick first ten ordered on last seen date